### ISSUE TITLE
local variable 'passfile' referenced before assignment

### ISSUE THREAD

**Issue by Nirajn2311** [thehappydinoa/iOSRestrictionBruteForce#5]
After your last commit, I am getting this error:
```
@Niraj MINGW64 ~/Desktop/VSCode/Python/iOSRestrictionBruteForce (master)
$ python ioscrack.py -a

 iOSRestrictionBruteForce
 Written by thehappydinoa


Looking for backups in C:\Users\Nandish\AppData\Roaming\Apple Computer\MobileSync\Backup\/... Directory Found
Found 9736e0d3a918a53bac6d16f1f647c435b44a41f5
Found 999de05a64dbb9c3a5b585b4965a3ac52ed1e2a8
Traceback (most recent call last):
  File "ioscrack.py", line 32, in <module>
    main()
  File "ioscrack.py", line 18, in main
    crackHashes(findHashes(BACKUP_PATHS))
  File "C:\Users\Nandish\Desktop\VSCode\Python\iOSRestrictionBruteForce\ioscrack\parse.py", line 17, in findHashes
    device = iDevice(path + bkup_dir)
  File "C:\Users\Nandish\Desktop\VSCode\Python\iOSRestrictionBruteForce\ioscrack\idevice.py", line 22, in __init__
    self.findSecretKeySalt()
  File "C:\Users\Nandish\Desktop\VSCode\Python\iOSRestrictionBruteForce\ioscrack\idevice.py", line 35, in findSecretKeySalt
    return self.findSecretKeySalt(newPath=(not newPath))
  File "C:\Users\Nandish\Desktop\VSCode\Python\iOSRestrictionBruteForce\ioscrack\idevice.py", line 37, in findSecretKeySalt
    line_list = passfile.readlines()
UnboundLocalError: local variable 'passfile' referenced before assignment
```
Also I use python 2.7

**Comment by thehappydinoa** [comment]
Ah that's my bad, I need to set passfile rather than returning it (idevice.py & line 37)

**Comment by thehappydinoa** [comment]
Fixed in f7e16e89106ea7998b3707e68f168ceadbdee726
